muacrypt/states.py

```diff
@@ -167,10 +168,11 @@ def update_from_msg(self, msg_id, effective_date, prefer_encrypt,
         if effective_date < self.autocrypt_timestamp:
             return
         if not keydata:
-            if effective_date > self.last_seen:
+            if effective_date >= self.last_seen:
```

Comment:
This way the same message may be appended as an noac_entry multiple times. Is that what we want?

Reply:
need to fix the tests indeed instead of this change which i will revert.